Ball_merge_surface_reconstruction/doc/Ball_merge_surface_reconstruction/Ball_merge_surface_reconstruction.txt

```diff
@@ -0,0 +1,77 @@
+
+The Local variant, however, is designed for handling open surfaces. As it is lightweight and requires only a single pass over the Delaunay simplices, it is optimal for reconstruction from an extremely huge point cloud.
+
+\subsection secBMSRPar Parameter Settings
+
+The main parameter used in BallMerge is the intersection ratio \f$ > \delta \f$. By definition, the \f$ > \delta \f$ ranges from 0 to 2 - with  \f$ BallMerge(0) \f being empty and  \f$ BallMerge(2) \f containing all the triangles of the Delaunay Triangulation. Experiments show that the \f$ > \delta \f$ value usually lies between 1.5 and 1.95 (with 1.8 or 1.85 giving the best results in most cases). Thanks to the nice property of this intersection ratio, it is quite easy to tune this parameter based on a simple rule: if the reconstructed surface has missing parts, the threshold should be lowered, or if the reconstructed surface has additional parts, the threshold should be increased. 
+The objective of the other parameter \f$ \eta\f$ is to remove unnecessarily long triangles from the result of Local BallMerge. In practice, the threshold \f$ \eta = 200\f$ was observed to perform well and can be adjusted - Figure \cgalFigureRef{figBMSRetatuning} shows the effect of varying \f$ \eta \f$.
+
+\cgalFigureBegin{figBMSRetatuning,BMSREtaTuning.png}
+Effect of \f$ \eta \f$ (\f$ \delta \f$ = 1.8); l.t.r.: \f$ \eta \f$ = 5,50,100,150,200,250,300 (original mesh taken from EPFL Statue Repository).
+\cgalFigureEnd
+
+\section secBMSRexamples Examples
+
+Two functions `ball_merge_surface_reconstruction_global()` and `ball_merge_surface_reconstruction_local()` are provided to use \f$ GlobalBallMerge \f$ and \f$ LocalBallMerge \f$ functionalities.
+
+`ball_merge_surface_reconstruction_global()` has the following arguments: The set of input points, two output parameters where the resulting meshes will be stored, and the parameter \f$ \delta \f$. The function returns two meshes - the largest and second-largest components as sometimes the expected shape will be the second-largest component after merging. A sample program showing how to use this function is given below:
+
+\cgalExample{Ball_merge_surface_reconstruction/ball_merge_reconstruction_global.cpp}
+
+`ball_merge_surface_reconstruction_local()` has the following arguments: The set of input points, an output parameter where the resulting mesh will be stored, the parameter \f$ \delta \f$, and the parameter \f$ \eta \f$ (to prune unnecessary long triangles). A sample program showing how to use this function is given below:
+
+\cgalExample{Ball_merge_surface_reconstruction/ball_merge_reconstruction_local.cpp}
+
+Alternatively, these functions can be combined as in the following program where argv[1] is the filename, argv[2] is the parameter \f$ \delta \f$, argv[3] decides whether to use local (0) or global variant (1), and arg[4] is an optional parameter \f$ \eta \f$ in case the user opts for the local variant.
```

Comment:
```suggestion
Alternatively, these functions can be combined as in the following program where `argv[1]` is the filename, `argv[2]` is the parameter \f$ \delta \f$, `argv[3]` decides whether to use local (0) or global variant (1), and `arg[4]` is an optional parameter \f$ \eta \f$ in case the user opts for the local variant.
```